tools.js

```diff
@@ -422,6 +429,38 @@ module.exports = (() => {
 			effect.toString = this.effectToString;
 			if (!effect.category) effect.category = 'Effect';
 			if (!effect.effectType) effect.effectType = 'Effect';
+
+			// Custom Game creation
+			if (customGame) {
+				effect = Object.clone(effect, true);
+				effect.id += customGame[2];
+				effect.isCustomGameFormat = true;
+				effect.cached = true;
+				delete effect.banlistTable;
+				effect.banlist = [];
+
+				let options = [
+					// Cartridge limitations
+					{option: 'Pokemon', type: 'ruleset'},
+					// Legal Pokemon
+					{option: ['Illegal', 'Unreleased'], type: 'banlist'},
+					// Team Preview
+					{option: 'Team Preview', type: 'ruleset'},
+					// Sleep Clause Mod
+					{option: 'Sleep Clause Mod', type: 'ruleset'},
+				];
+
+				customGame = parseInt(customGame[2]);
+				for (let i = 0; i < options.length; i++) {
+					if (!(customGame & (1 << i))) continue;
```

Comment:
Feels strange seeing bit level masking for flags when we've always done it with explicit flag names